command/ca/certificate.go

```diff
@@ -266,10 +267,11 @@ func (f *certificateFlow) getClient(ctx *cli.Context, subject, token string) (ca
 	return ca.NewClient(caURL, options...)
 }
 
+// GenerateToken generates the token using the token flow or the online mode.
```

Comment:
"offline token flow"?

Reply:
GenerateToken generates a token for immediate use (therefore only default validity values will be used). The token is generated either with the offline token flow or the online mode.

Reply:
offline flow, yes